django_ses/views.py

```diff
@@ -191,15 +191,15 @@ def handle_bounce(request):
         notification = json.loads(raw_json.decode('utf-8'))
     except ValueError as e:
         # TODO: What kind of response should be returned here?
-        logger.warning('Recieved bounce with bad JSON: "%s"', e)
+        logger.warning(u'Recieved bounce with bad JSON: "%s"', e)
```

Comment:
Looks like this should say `Received` everywhere. Mind updating while you're here? :)